web/src/ResourceStatusSummary.tsx

```diff
@@ -283,31 +283,38 @@ type ResourceStatusSummaryProps = {
 }
 
 export function ResourceStatusSummary(props: ResourceStatusSummaryProps) {
-  // Count the statuses.
+  // Count and calculate the combined statuses.
   let resources = props.view.uiResources || []
+  const allStatuses: ResourceStatus[] = []
 
   let testResources = new Array<UIResource>()
+  const testStatuses: ResourceStatus[] = []
```

Comment:
i wonder if it would make sense to rip out the test stuff first, so you don't have to stress about it.

Reply:
that's not a bad idea at all! how about i merge this one in as-is and then address that ticket next?

Reply:
:+1: